components/xsd-fu/templates-cpp/OMEXMLModelObject.template

```diff
@@ -1,12 +1,3 @@
-{% python
-  def getConcreteClasses(abstractClass):
-    returnList = []
-    if model.opts.lang.hasSubstitutionGroup(abstractClass):
-      for o in sorted(model.objects.values(), lambda x, y: cmp(x.name, y.name)):
-          if o.parentName == abstractClass:
-            returnList.append(o.name)  
-    return returnList
```

Comment:
I had made a previous comment about removing getConcreteClasses from property, but now I see you've moved it to there from the template. In which case both changes are fine to leave as they are